client/app/config/EU/common.constants.js

```diff
@@ -234,6 +234,10 @@ angular.module("managerApp")
             veeam: {
                 FR: "https://www.ovh.com/fr/storage-solutions/veeam-cloud-connect/"
             },
+            veeam_enterprise: {
+                // TODO: To update when website is prodded
```

Comment:
Well users won't be able to use the manager before ordering the product I guess :)